ASSUMPTIONS.md

```diff
@@ -99,3 +102,87 @@ As more statistical data is collected about slashing in different validator envi
+
+To change this value, update the `date_eip1559` [System Parameter](./model/system_parameters.py).
+
+#### Proof-of-Stake Activation Date ("The Merge")
+
+The Ethereum researcher Justin Drake estimates a "best guess" of 1 December 2021 for the Proof-of-Stake activation date in his [ETH supply analysis](https://docs.google.com/spreadsheets/d/1ZN444__qkPWPjMJQ_t6FfqbhllkWNhHF-06ivRF73nQ). Launch dates are hard to predict, and for more rigorous analysis a range of values should be tested.
+
+To change this value, update the `date_pos` [System Parameter](./model/system_parameters.py).
+
+## Ethereum Transaction & EIP-1559 Assumptions
+
+### Average Gas Used
+
+With the introduction of [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) the current gas limit is replaced by two values:
+* a “long-term average target” equal to the current gas limit
+* a “hard per-block cap” which is twice the current gas limit
+
+The long-term average gas target per block is set to 15m gas; by default we assume the gas used per block will on average be equal to the gas target. To change this value, update the `gas_target_process` [System Parameter](./model/system_parameters.py).
+
+### Average Fee Cap (`max_fee_per_gas`)
+
+The [EIP-1559 proposal](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) defines a variable `max_fee_per_gas` to set a fee cap per transaction. This fee cap is the maximum fee in Gwei per gas a user will pay per transaction. This fee is made up of the base fee and a priority fee, where the base fee is burned and the priority fee is paid to miners/validators.
+
+Pre EIP-1559 the fee cap is equivalent to the gas price in Gwei per gas. We use the historical mean gas price over the past 12 months to set the average fee cap, which in this model is equivalent to the sum of the values from the `base_fee_process` and `priority_fee_process`.
+
+We use a static value of 100 Gwei per gas (historical mean gas price) to set the default values for the `base_fee_process` and `priority_fee_process`. See [Miner Extractable Value as Percentage of Average Fee Cap](#miner-extractable-value-as-percentage-of-average-fee-cap) for further related assumptions.
+
+### Miner Extractable Value as Percentage of Average Fee Cap
+
+TODO: get Roger to review and validate
+
+Miner Extractable Value (MEV) is a measure of the profit a miner/validator can make by arbitrarily including, excluding, or re-ordering transactions within the blocks they produce.
+
+Under the influence of MEV, it is assumed a certain proportion of miners/validators would run MEV nodes (e.g. Flashbots), resulting in a blockspace auction where users would need to increase their transaction priority fee in order to get transactions included in the blockspace. In steady state, without the influence of MEV, it is assumed the priority fee would be on average 1 Gwei per gas to compensate for uncle risk - see https://notes.ethereum.org/@vbuterin/BkSQmQTS8#Why-would-miners-include-transactions-at-all.
```

Comment:
The node software that is run by miners to benefit from MeV is called mev-geth ( https://github.com/flashbots/mev-geth ),written by Flashbots. There are also other implementations that are run, but they are not publicly accessible. 
The priority fee applies to EIP-1559 and not MeV. MeV transactions get priority over any other transactions and they use a separate payment channel to gas fee or priority fee. With Flashbots, there are seekers that submit transaction bundles to Miner's off chain, and the profit that the transactions in that bundle produce is shared with the miners.